models/tmp/stg_iterable__user_unsub_message_type_history_tmp.sql

```diff
@@ -0,0 +1,4 @@
+{{ config(enabled=var('iterable__using_user_unsubscribed_message_type_history', True)) }}
+
+select * from {{ var('user_unsubscribed_message_type_history') }}
+-- had to rename this to be compatible with postgres....
```

Comment:
Did this need to be renamed? What error was persisting for only Postgres?

Reply:
Super tiny, can you update the format to be like the below:
```sql
select *
from {{ var('table_name') }}
```
Just to keep with consistency across packages.

Reply:
yeah postgres is the only one that maxes table names out at like 64, and it looks like postgres appends weird little suffixes to the models as they're being run/tested? 
https://app.circleci.com/pipelines/github/fivetran/dbt_iterable_source/64/workflows/31be6ae3-266d-4e3e-ba94-a8de26490407/jobs/64/parallel-runs/0/steps/0-105

Reply:
Ah that is interesting and also strange?! I will add this to our core process docs to make sure we have a max model name length so we do not encounter issues like this in the future.